Floofbot/Modules/Tag.cs

```diff
@@ -7,191 +7,188 @@
                 }
             }
-
-            [Command("add")]
-            [Priority(1)]
-            public async Task Add()
-            {
-                await sendEmbed(createDescriptionEmbed($"💾 Usage: `tag add [name] [content]`"));
+            catch (SqliteException) {
+                await SendEmbed(CreateDescriptionEmbed($"💾 Tag `{Tag}` Already Exists"));
             }
+        }
 
-            //TODO: Fix me
-            [Command("list")]
-            public async Task ListTags([Remainder] string Content = null)
-            {
-                string guildId = $"{Context.Guild.Id}";
-                string sql = $"SELECT tagID FROM Tags WHERE tagID LIKE '%:$guildId%'";
-                SqliteCommand command = new SqliteCommand(sql, dbConnection);
-                command.Parameters.Add(new SqliteParameter("$guildId", guildId));
-                var result = command.ExecuteReader();
+        [Command("add")]
+        [Priority(1)]
+        public async Task Add()
+        {
+            await SendEmbed(CreateDescriptionEmbed($"💾 Usage: `tag add [name] [content]`"));
+        }
 
-                List<string> tags = new List<string>();
-                List<string> pages = new List<string>();
+        //TODO: Fix me
+        [Command("list")]
+        public async Task ListTags([Remainder] string Content = null)
+        {
+            string guildId = $"{Context.Guild.Id}";
+            string sql = $"SELECT tagID FROM Tags WHERE tagID LIKE '%:$guildId%'";
+            SqliteCommand command = new SqliteCommand(sql, dbConnection);
+            command.Parameters.Add(new SqliteParameter("$guildId", guildId));
+            var result = command.ExecuteReader();
 
-                while (result.Read()) {
-                    tags.Add(result.GetString(0).Split(':')[0]);
-                }
+            List<string> tags = new List<string>();
+            List<string> pages = new List<string>();
+
+            while (result.Read()) {
+                tags.Add(result.GetString(0).Split(':')[0]);
+            }
 
-                tags = tags.OrderBy(x => x).ToList();
-                int index = 0;
-                for (int i = 1; i <= (tags.Count / 50) + 1; i++) {
-                    string text = "```glsl\n";
-                    int pagebreak = index;
-                    for (; index < pagebreak + 50; index++) {
-                        if (index < tags.Count) {
-                            text += $"[{index}] - {tags[index].ToLower()}\n";
-                        }
+            tags = tags.OrderBy(x => x).ToList();
+            int index = 0;
+            for (int i = 1; i <= (tags.Count / 50) + 1; i++) {
+                string text = "```glsl\n";
+                int pagebreak = index;
+                for (; index < pagebreak + 50; index++) {
+                    if (index < tags.Count) {
+                        text += $"[{index}] - {tags[index].ToLower()}\n";
```

Comment:
this seems a bit messy, from what I gather, couldnt you just use foreach?

Reply:
This part will be handled by a future PR.